### PR TITLE
adding subresources/status to imagepruner crd

### PR DESCRIPTION
**Pull request by coreydaley**

### CODE REVIEW

File: imageregistry/v1/01-crd.yaml [hunk 1]
```diff
@@ -7,6 +7,8 @@ spec:
   scope: Cluster
   version: v1
   preserveUnknownFields: false
+  subresources:
+    status: {}
```
Comment (adambkaplan) [inline imageregistry/v1/01-crd.yaml]: @sttts @soltysh would adding the `+kubebuilder:subresource:status` tag to the object generate this for us automatically?

Comment (soltysh) [reply imageregistry/v1/01-crd.yaml]: We're not running full generation, so this won't help, but it worthy adding in the nearest future.

**Pull request merged**